daemon/mgr/cri.go

```diff
@@ -312,7 +327,10 @@ func (c *CriManager) RemovePodSandbox(ctx context.Context, r *runtime.RemovePodS
 		return nil, fmt.Errorf("failed to remove root directory %q: %v", sandboxRootDir, err)
 	}
 
-	c.SandboxStore.Remove(podSandboxID)
+	err = c.SandboxStore.Remove(podSandboxID)
+	if err != nil {
```

Comment:
Should we return err info to Caller, if we do not deal with the error, there will have more and more dirty data, WDYT?

Reply:
Agree :)